lib/server/server-main.js

```diff
@@ -2,3 +2,7 @@
 import './../imports/startup/server/server-index';
 
 import './../imports/api/documents/both/methods';
+
+import { ProseMeteorServer as ProseMeteorServerClass } from './../imports/api/prosemeteor/server/prosemeteor-server';
+
+export let ProseMeteorServer = ProseMeteorServerClass;
```

Comment:
thanks, I was looking around a ton of es2015 docs to try to find the correct syntax for that and couldn't find it anywhere